dist/justgage.js

```diff
@@ -838,8 +838,20 @@
     var obj = this;
     var displayVal, color;
 
-    max = max || null;
-    min = min || null;
+    if (Number.isInteger(max) || Number(max) === max && max % 1 !== 0) {
```

Comment:
I would create a custom function for this check and use something like

```
function isNumber(n) {
  return n !== null && n !== undefined && !isNaN(n)
}
```

and then use it like:

```
max = isNumber(max) ? max : null
min = isNumber(min) ? min : null
```